examples/raspi/src/main.rs

```diff
@@ -0,0 +1,83 @@
+use embedded_graphics::fonts::Font6x8;
+use embedded_graphics::image::Image;
+use embedded_graphics::pixelcolor::BinaryColor;
+use embedded_graphics::prelude::*;
+use embedded_graphics::primitives::{Circle, Line, Rectangle};
+use embedded_graphics::Drawing;
+use linux_embedded_hal::I2cdev;
+use machine_ip;
+use ssd1306::{mode::GraphicsMode, Builder};
+use std::thread::sleep;
+use std::time::Duration;
+extern crate ctrlc;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+
+fn main() {
+    let running = Arc::new(AtomicBool::new(true));
+    let r = running.clone();
+    ctrlc::set_handler(move || {
+        r.store(false, Ordering::SeqCst);
+    })
+    .expect("Error setting Ctrl-C handler");
+
+    let i2c = I2cdev::new("/dev/i2c-1").unwrap();
+
+    let mut disp: GraphicsMode<_> = Builder::new().connect_i2c(i2c).into();
```

Comment:
In new 0.4.0 version Builder have not connect_i2c method.

Reply:
ah yes that's a good point. `examples/raspi/cargo.toml` uses 0.3.something. I should update that too.

Reply:
This should be fixed. I updated a bunch of dependencies and worked around the breaking changes